azure-pipelines.yml

```diff
@@ -114,16 +114,9 @@ stages:
             python3 -m pip install -r test_requirements.txt && \
             echo CFLAGS \$CFLAGS && \
             python3 -m pip install -v . && \
-            python3 runtests.py -n --debug-info --mode=full -- -rsx --junitxml=junit/test-results.xml && \
-            python3 -m pip install threadpoolctl && \
-            python3 tools/openblas_support.py --check_version"
+            cd tools && \
+            python3 -m pytest --pyargs numpy"
```

Comment:
This has removed the check for the openblas version. Is that on purpose?

Reply:
Yes, we now have proper logging in the build log for the version, and this cannot really be wrong (no other OpenBLAS version is installed in CI). This job was painful to debug, so I trimmed it down as much as possible.